packages/coinstac-ui/app/render/components/consortium-card.js

```diff
@@ -25,7 +25,7 @@ export default function ConsortiumCard(props) {
 
   if (isOwner) {
     deleteButton = (
-      <Button bsSize="small" bsStyle="danger" onClick={deleteConsortium}>
+      <Button bsSize="small" bsStyle="danger" onClick={deleteConsortium} id={`delete-${label}`}>
```

Comment:
A consortium’s `label` may have whitespace, which means selection by ID could break. Perhaps change to the consortium’s `id`?